less/admin/ExtensionWidget.less

```diff
@@ -0,0 +1,86 @@
+.ExtensionsWidget {
+  background-color: @body-bg;
+  padding: 0;
+}
+
+.ExtensionsWidget-list {
+  > .container {
+    padding: 0;
+    background-color: @body-bg;
+
+    .ExtensionList-Category {
+      background: @control-bg;
+      padding: 20px 0 30px 20px;
+      margin-bottom: 20px;
+      border-radius: @border-radius;
+
+      .ExtensionList-Label {
+        margin-top: 0;
+        color: @muted-color;
+      }
+    }
+
+    .ExtensionGroup {
+      margin-bottom: 20px;
+
+      h3 {
+        color: @muted-color;
+        text-transform: uppercase;
+        font-size: 12px;
+        margin: 0 0 10px;
+      }
+    }
+
+    .ExtensionList {
+      margin: 0 -6px;
+      padding: 0;
+      list-style: none;
+      display: grid;
+      grid-gap: 10px;
+      grid-template-columns: repeat(auto-fit, 90px);
+
+      > li {
+        text-align: left;
+        position: relative;
+        display: block;
+      }
+    }
+  }
+
+  .ExtensionListItem.disabled {
+    .ExtensionListItem-title {
+      opacity: 0.5;
+      color: @muted-color;
+    }
+
+    .ExtensionListItem-icon {
+      opacity: 0.5;
+    }
+  }
+
+  .ExtensionListItem {
+    width: 90px;
+    height: 90px;
+    margin: 6px;
```

Comment:
Same here, we can remove the margins since we're using grid

Also we should remove the height, since we are now showing the extension name

Reply:
Also let's set `text-decoration` to none when the anchor inside the `ExtensionListItem` is hovered, it doesn't look good with underline.

Which makes me wonder if we want to add any kind of hovering effect, not sure yet.

Reply:
I dropped the width & margin but left in the height because it tightens it up, reduces the extra spacing. For the hovering effect, I added a small scale increase of the icon and title.

Reply:
the problem with setting the height is the text is hidden though

![flarum lan_admin (1)](https://user-images.githubusercontent.com/20267363/98725866-fc9a4500-2395-11eb-935c-61d6f1f63ebe.png)